bayes_opt/acquisition.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, random_state=None):
     @abc.abstractmethod
     def base_acq(self, *args, **kwargs):
         """Provide access to the base acquisition function."""
-        pass # pragma: no cover
+        # pragma: no cover
```

Comment:
is excluding this line from coverage still necessary considering there's no actual statement made here?

Reply:
After adding the rule to remove `pass`, I forgot to add a comment. I'll fix it.